### PR TITLE
#129 PsCookie doesn't set HttpOnly

### PR DESCRIPTION
**Pull request by dmzaytsev**
task #129 
added HttpOnly flag to cookie produced by PsCookie 
updated unit test

### CODE REVIEW

**Comment by davvd** [comment]
Thanks, I will ask someone to do a code review here, thanks for your pull request

**Comment by davvd** [comment]
@pinaf please review, thanks

File: src/main/java/org/takes/facets/auth/PsCookie.java [hunk 1]
```diff
@@ -125,7 +125,8 @@ public Response exit(final Response res,
                     System.currentTimeMillis()
                         + TimeUnit.DAYS.toMillis(this.age)
                 )
-            )
+            ),
+            "HttpOnly"
```
Comment (pinaf) [inline src/main/java/org/takes/facets/auth/PsCookie.java]: @dmzaytsev let's add `"HttpOnly"` before `Expires`, on line 121.

File: src/test/java/org/takes/facets/auth/PsCookieTest.java [hunk 2]
```diff
@@ -51,8 +51,11 @@ public void addsCookieToResponse() throws IOException {
                     new CcPlain(), "foo", 1L
                 ).exit(new RsEmpty(), new Identity.Simple("urn:test:99"))
             ).print(),
-            Matchers.containsString(
-                "Set-Cookie: foo=urn%3Atest%3A99;Path=/;"
+            Matchers.allOf(
+                Matchers.containsString(
+                    "Set-Cookie: foo=urn%3Atest%3A99;Path=/;"
+                ),
+                Matchers.containsString(";HttpOnly")
```
Comment (pinaf) [inline src/test/java/org/takes/facets/auth/PsCookieTest.java]: @dmzaytsev Now we can incorporate this into the string above - no need for `Matchers.allOf`
```java
Matchers.containsString(
    "Set-Cookie: foo=urn%3Atest%3A99;Path=/;HttpsOnly"
)
```

**Comment by pinaf** [comment]
@dmzaytsev looks good - 2 minor comments above.

**Comment by dmzaytsev** [comment]
@pinaf check the changes please

**Comment by pinaf** [comment]
@dmzaytsev thanks!

**Comment by pinaf** [comment]
@rultor merge

**Comment by yegor256** [comment]
@rultor merge this

**Comment by davvd** [comment]
@elenavolokhova please, review this ticket for compliance with our [QA rules](http://www.teamed.io/qa.html)

**Comment by davvd** [comment]
@rultor please deploy

**Comment by elenavolokhova** [comment]
> davvd commented 3 days ago
Thanks, I will ask someone to do a code review here, thanks for your pull request

Please don't forget to address your comments in the future.

**Comment by elenavolokhova** [comment]
@pinaf  According to our [quality rules](http://www.teamed.io/qa.html): 
> The code reviewer found at least three problems in the code. 

Only two issues found during review. Please, try to find at least three problems next time.

**Comment by elenavolokhova** [comment]
@davvd Ticket passed QA review. Quality is acceptable here.

**Comment by davvd** [comment]
> @davvd Ticket passed QA review. Quality is acceptable here.

@elenavolokhova thanks a lot, next time everybody should try to make it better

**Comment by davvd** [comment]
@pinaf I added 10 mins to @elenavolokhova (for QA review) in transaction 55065571... Thanks a lot, I just topped your account for **18 mins**, transaction ID `AP-9FV10622WP145404N`, total time was 22 hours and 49 mins... extra minutes for review comments (c=3)... +18 added to your rating, at the moment it is: [+6234](http://www.netbout.com/b/35128?open=rating)

**Pull request merged**